piexif/_load.py

```diff
@@ -153,7 +153,8 @@ def convert_value(self, val):
         elif t == TYPES.Ascii: # ASCII
             if length > 4:
                 pointer = struct.unpack(self.endian_mark + "L", value)[0]
-                data = self.tiftag[pointer: pointer+length - 1]
+                data = self.tiftag[pointer: pointer+length]
+                data = data.split(b'\x00', 1)[0]
             else:
                 data = value[0: length - 1]
```

Comment:
This case is uncovered, `data = data.split(b'\x00', 1)[0]` should be after `if` statement.